src/modules/physics/physics_jolt.c

```diff
@@ -1975,6 +1976,8 @@ void lovrDistanceJointGetLimits(DistanceJoint* joint, float* min, float* max) {
 }
 
 void lovrDistanceJointSetLimits(DistanceJoint* joint, float min, float max) {
+  lovrCheck(min <= 0.f, "Distance joint lower distance limit must be negative");
```

Comment:
min can be positive, it just needs to be <= max (e.g. min distance of 1 will prevent colliders from getting closer than 1m, min == max maintains constant distance).